Source/WebInspectorUI/UserInterface/Controllers/HARBuilder.js

```diff
@@ -371,18 +371,16 @@ WI.HARBuilder = class HARBuilder
     static protocolFromHARProtocol(protocol)
     {
         switch (protocol) {
+        case "HTTP/3":
+            return "h3";
         case "HTTP/2":
             return "h2";
-        case "HTTP/1.0":
```

Comment:
yeah we should keep these until Web Inspector drops support for systems that use it

it's pretty low cost to keep around, so id rather err on the side of caution

Reply:
I think there was previously some misunderstanding of the URLSession API. The `protocol` comes from [URLSessionTaskTransactionMetrics.networkProtocolName](https://developer.apple.com/documentation/foundation/urlsessiontasktransactionmetrics/1643141-networkprotocolname) which is the ALPN of the connection. `http/1.0` was never a valid ALPN.

The 3 possible values are `http/1.1`, `h2`, and `h3`. In the case of HTTP/1, CFNetwork always sends an HTTP/1.1 request. The server might decide to respond with an HTTP/1.0 or HTTP/0.9 response, but that doesn't affect the ALPN value which is negotiated through TLS (it defaults to `http/1.1` if the server does not pick one).

Reply:
is that true for all ports of WebKit?  if so then yes i agree they should be removed.

Reply:
I'm not an expert on other ports of WebKit. If the `protocol` always comes from the ALPN value, `http/1.0` shouldn't be possible since it's not a standard ALPN.